nx-X11/lib/X11/Pending.c

```diff
@@ -24,7 +24,9 @@ in this Software without prior written authorization from The Open Group.
 
 */
 
+#if defined(NX_TRANS_SOCKET) && defined(NX_TRANS_TEST)
 #include <stdio.h>
+#endif
```

Comment:
Shouldn't this go into the libX11 -> 1.3.4 PR? If so, please add it there with our current iteration.